src/status_im/contexts/wallet/send/select_address/tabs/view.cljs

```diff
@@ -20,22 +39,8 @@
         :container-style style/empty-container-style}]
       [rn/view {:style style/my-accounts-container}
        (doall
-        (for [{:keys [color address] :as account} other-accounts]
-          ^{:key (str address)}
-          (let [transformed-address (rf/sub [:wallet/account-address address
-                                             (:network-preferences-names account)])]
-            [quo/account-item
-             {:account-props (assoc account
-                                    :customization-color color
-                                    :address             transformed-address
-                                    :full-address?       true)
-              :on-press      #(rf/dispatch [:wallet/select-send-address
-                                            {:address   address
-                                             :recipient {:recipient-type      :account
-                                                         :label               (:name account)
-                                                         :customization-color (:color account)
-                                                         :emoji               (:emoji account)}
-                                             :stack-id  :screen/wallet.select-address}])}])))])))
+        (for [{:keys [address] :as account} other-accounts]
+          ^{:key (str address)} [other-account-item account]))])))
```

Comment:
This PR also solves a warning we had about repeated react keys in this screen. This is a refactor and a solution for it